chart/templates/mongodb-statefulset.yaml

```diff
@@ -54,11 +54,12 @@ spec:
         - name: mongodb
           containerPort: 27017
         volumeMounts:
-        - name: data
+        - name: mongodb-pv-storage
           mountPath: /bitnami/mongodb
           subPath: 
         resources:
           {}
       volumes:
```

Comment:
Should we use a volumeclaimtemplate?https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components

Reply:
Ah yes! But now I'm wondering if we should go that route because that would involve explaining StatefulSets which we decided to skip since this was more for beginners 🤔 should we maybe let it be as it were?

Reply:
@RinkiyaKeDad I think we should use a proper PVC in our main sample. I was planning to do that, but Cloud was super slow creating volumes until now because we were hitting a CSI-driver bottleneck.
Another option is to have a dedicated repo/sample for your tutorials. For example, I think Helm at this level, makes things harder

Reply:
Totally agree with that! Let me change this PR to add a PVC for the mongodb stateful set and for the article I'll not link it with the movies app for now and show some standard YAMLs from the docs, how does that sound? 

For future series, we'll make sure we have all the necessary stuff in a separate repo beforehand :)

Reply:
Thanks!